src/drivers/imu/invensense/icm42688p/ICM42688P.hpp

```diff
@@ -179,9 +179,10 @@ class ICM42688P : public device::SPI, public I2CSPIDriver<ICM42688P>
 	int32_t _fifo_gyro_samples{static_cast<int32_t>(_fifo_empty_interval_us / (1000000 / GYRO_RATE))};
 
 	uint8_t _checked_register_bank0{0};
-	static constexpr uint8_t size_register_bank0_cfg{16};
+	static constexpr uint8_t size_register_bank0_cfg{17};
 	register_bank0_config_t _register_bank0_cfg[size_register_bank0_cfg] {
 		// Register                              | Set bits, Clear bits
+		{ Register::BANK_0::INTF_CONFIG0,         INTF_CONFIG0_BIT::UI_SIFS_CFG, 0}, // 0x3 disable I2C
```

Comment:
Can you put this just before INTF_CONFIG1 (0x4D) to keep it in order?

Reply:
Yes, I intended it to be as early as possible, since it protects the bus. But it doesn't make much difference, it looks better like you suggested. Done